systest/multi-tenancy/basic_test.go

```diff
@@ -37,6 +37,11 @@ import (
 	"github.com/dgraph-io/dgraph/x"
 )
 
+type inputTriplets_Count struct {
+	lower_limit int
```

Comment:
IIRC, we're following the Google Go style guide. In which case these should be camelCase and not underscore... https://google.github.io/styleguide/go/guide#mixedcaps

Reply:
done @matthewmcneely